tests/test_local_regular_grid.py

```diff
@@ -0,0 +1,227 @@
+import pandas as pd
+import pytest
+
+from polytope.engine.hullslicer import HullSlicer
+from polytope.polytope import Polytope, Request
+from polytope.shapes import Point, Select
+
+
+class TestSlicingFDBDatacube:
+    def setup_method(self, method):
+        from polytope.datacube.backends.fdb import FDBDatacube
+
+        # Create a dataarray with 3 labelled axes using different index types
+        self.options = {
+            "values": {
+                "mapper": {
+                    "type": "local_regular",
+                    "resolution": [80, 80],
+                    "axes": ["latitude", "longitude"],
+                    "local": [-40, 40, -20, 60],
+                }
```

Comment:
As discussed, here we are assuming the local grid's first point is aligned with the bounds we put here. This might not always be the case, if the local grid was created by cropping a global grid.

![image](https://github.com/ecmwf/polytope/assets/11695815/7431ea12-f74b-4853-9303-92555aaf7438)

When we have some real grids to try we might need to change this behaviour.